behave/features/steps/regex_steps.py

```diff
@@ -79,3 +119,16 @@ def step_impl_means_test_result(context, optional):
 def step_impl_popup_button(context, optional):
     modal = context.helperfunc.find_by_css_selector(".modal-dialog")
     modal.find_element_by_xpath("//button[@type='submit']").click()
+
+
+@step("the green tick is (?P<optional>not )?present in the Finance tab")
+def step_impl_green_tick_present(context, optional):
+    tabs = context.helperfunc.find_by_css_selector("ul.Tabs")
+    finance_tab_link = tabs.find_element_by_link_text("Finances")
+    classes = finance_tab_link.get_attribute("class")
+    if optional:
```

Comment:
This may need wait functionality and could be used elsewhere.
https://github.com/ministryofjustice/cla-end-to-end-tests/blob/0b94ebd3b16d98409a10a95260d113df11ea431f/behave/features/steps/frontend_specialist_provider.py#L314

https://github.com/ministryofjustice/cla-end-to-end-tests/blob/0b94ebd3b16d98409a10a95260d113df11ea431f/behave/features/steps/frontend_assignment.py#L296

Reply:
I think I've done this now...but might need confirmation in the morning I've done what you meant 🤔

Reply:
Looking at [this ](https://github.com/ministryofjustice/cla-end-to-end-tests/blob/0b94ebd3b16d98409a10a95260d113df11ea431f/behave/features/steps/frontend_assignment.py#L296) I think the `wait until` functionality is not required as it is actually just delaying a move to a new tab.
I like how you have used one function for both steps BUT here I was sort of expecting a function something along the lines of 

`def finances_approved(context,classes): 
return "Icon--solidTick" in classes and "Icon--green" in classes
` 
which could then be used in `frontend_specialist_provider.py ` and `frontend_assignment.py` but not essential.